code/API_definitions/simple-edge-discovery.yaml

```diff
@@ -1,718 +1,714 @@
+    description: |
+      Find the closest Edge Cloud Zone to the user device.
 
 paths:
- /edge-cloud-zones:
-  get:
-   security:
-    - openId:
-       - simple-edge-discovery:edge-cloud-zones:read
-   operationId: readClosestEdgeCloudZone
-   parameters:
-    - name: filter
-      in: query
-      required: true
+  /edge-cloud-zones:
+    get:
+      security:
+        - openId:
+            - simple-edge-discovery:edge-cloud-zones:read
+      operationId: readClosestEdgeCloudZone
+      parameters:
+        - name: filter
+          in: query
+          required: true
+          description: |
+            Filter the Edge Cloud Zones according to the parameter value.
+            For this API the only supported value is `closest`.
+          schema:
+            type: string
+            enum:
+              - closest
+
+        - name: IPv4-Address
+          in: header
+          required: false
+          description: The public IPv4 address allocated to the device by the network operator.
+          example: "84.125.93.10"
+          schema:
+            $ref: "#/components/schemas/SingleIpv4Addr"
+
+        - name: Public-port
+          in: header
+          required: false
+          description: The public TCP or UDP port allocated to the device by the network operator.
+          example: 123
+          schema:
+            $ref: "#/components/schemas/Port"
+
+        - name: IPv6-Address
+          in: header
+          required: false
+          description: The public IPv6 address allocated to the device by the network operator.
+          example: "2001:db8:85a3:8d3:1319:8a2e:370:7348"
+          schema:
+            $ref: "#/components/schemas/DeviceIpv6Address"
+
+        - name: Network-Access-Identifier
+          in: header
+          required: false
+          description: |
+            3GPP network access identifier for the subscription
+            being used by the device.
+          schema:
+            $ref: "#/components/schemas/NetworkAccessIdentifier"
+
+        - name: Phone-Number
+          in: header
+          example: "+441234567890"
+          required: false
+          description: |
+            MSISDN in E.164 format (starting with country code) of
+            the mobile subscription being used by the device. Optionally
+            prefixed with '+'.
+          schema:
+            $ref: "#/components/schemas/PhoneNumber"
+
+        - name: x-correlator
+          in: header
+          required: false
+          description: |
+            When the API Consumer includes the "x-correlator" header in the
+            request, the API provider must include it in the response with
+            the same value that was used in the request. Otherwise, it is
+            optional to include the "x-correlator" header in the response with
+             any valid value. Recommendation is to use UUID for values.
+          schema:
+            type: string
+            pattern: ^[a-zA-Z0-9-]{0,55}$
```

Comment:
You have a definition within the components section ... would recommend to use that (and add the pattern there)

Reply:
Done (per previous comment)